{code}
fun foo(e: VcsException) {
  val f = "needs resolve" in e.message
}{code}

{code}
import org.jetbrains.annotations.NotNull;

public class VcsException extends Exception {
  @Override
  @NotNull
  public String getMessage() {
    return "";
  }
}
{code}

    Operator call corresponds to a dot-qualified call 'e.message.contains("needs resolve")' which is not allowed on a nullable receiver 'e,message'